index.js

```diff
@@ -421,6 +422,10 @@ function slowToString (encoding, start, end) {
   }
 }
 
+// Even though this property is private, it shouldn't be removed because it is
+// used by `is-buffer` to detect buffer instances in Safari 5-7.
+Buffer.prototype._isBuffer = true
```

Comment:
It was already always set (since it's [added inside `Buffer._augment`](https://github.com/feross/buffer/blob/master/index.js#L1328) which was always called). If it wasn't the case then `Buffer.isBuffer` would've failed since it [uses this property](https://github.com/feross/buffer/blob/master/index.js#L269). I just moved it to the prototype to avoid having to add it everywhere.